### PR TITLE
Venu choudhary/cord

### PR DESCRIPTION
**Pull request by VenuChoudhary001**
## CORD NETOWRK : WEEK 6 UPDATES

- Worked on using APIs provided by sunbird rc.
- Testing and troubleshooting API endpoints.
- Backend code integration with issuer agent.

### CODE REVIEW

**Comment by Vivek-Kumar-Shrivastav** [comment]
### **Week-5**
1. In the milestones you have to write what you stages you have achieved in the entire week. Try to add those and not how it is done.
2. Add the link of project PR that you have raised in Contribution section
3. Add these things in Learning section, whatever new thing you have learnt

File: docs/2023/CORD Network/IntegrationWithSunbirdRC/updates/2023-08-11.md [hunk 1]
```diff
@@ -5,13 +5,41 @@ author: Venu Choudhary
...
 
-## Contributions
+Sunbird RC provides us with a set of autogenerated APIs which is one of the USP of this plaform. Now in order to use these APIs, most of the times we need to be authenticated, and in the previous week we have already made the keycloak setup,the authentication provider for Sunbird.
+
+During this week, I worked on the schema creation APIs. [Links](https://docs.sunbirdrc.dev/api-reference/schema/create-schema)
+
+Now, when we hit this endpoint, we need to make sure we are authorized. We will need to authenticate as the admin user we created previous week, along with the access token.
+
+```bash
+    http://keycloak:8080/auth/realms/sunbird-rc/protocol/openid-connect/token
```
Comment (Vivek-Kumar-Shrivastav) [inline docs/2023/CORD Network/IntegrationWithSunbirdRC/updates/2023-08-11.md:19]: This is not working. May you mention what is this for.

**Comment by Vivek-Kumar-Shrivastav** [comment]
### **Week-6**

1. Raise the PR and add it in the contribution section.
2. Things you learn in Learning Section.

File: docs/2023/CORD Network/IntegrationWithSunbirdRC/updates/2023-08-11.md (hunk 1 above)
Comment (VenuChoudhary001) [reply docs/2023/CORD Network/IntegrationWithSunbirdRC/updates/2023-08-11.md:19]: This is basically for authenticating users. When you hit this endpoint, we get an access token.

**Comment by Vivek-Kumar-Shrivastav** [comment]
**Error:  Client bundle compiled with errors therefore further build is impossible.**

**Pull request merged**